CorsixTH/Lua/hospital.lua

```diff
@@ -1030,7 +1033,7 @@ function Hospital:onEndDay()
   if breakdown == 1 and not self.heating_broke and self.boiler_can_break and
       self.world.object_counts.radiator > 0 then
     if tonumber(self.world.map.level_number) then
-      if self.world.map.level_number == 1 and (self.world.month > 5 or self.world.year > 1) then
+      if self.world.map.level_number == 1 and (self.world:date():monthOfYear() > 5 or self.world:date():year() > 1) then
```

Comment:
Could be simplified to `self.world:date() > Date(1,5,1)` there are several places like this.

Reply:
Done, could you please also point me to other places ? I found some other comparisons as well but they are mostly about some part of year but every year so I couldn't figure out how to compare them.